gallery/src/components/NxBreadcrumb/NxBreadcrumbPage.tsx

```diff
@@ -0,0 +1,219 @@
+              <NxTable.Cell>
+                <NxCode>NxBreadcrumb</NxCode> supports any HTML attribute that's normally
+                supported by <NxCode>&lt;nav&gt;</NxCode> elements.
+              </NxTable.Cell>
+            </NxTable.Row>
+          </NxTable.Body>
+        </NxTable>
+      </NxTile.Subsection>
+      <NxTile.Subsection>
+        <NxTile.SubsectionHeader>
+          <NxH3>Semantics and Accessibility</NxH3>
+        </NxTile.SubsectionHeader>
+        <NxP>
+          <NxCode>NxBreadcrumb</NxCode> renders as a <NxCode>&lt;nav&gt;</NxCode> element with its default role
+          of <NxCode>navigation</NxCode>. It has a default accessible name of "breadcrumbs" which may be overridden
+          by specifying an alternative <NxCode>aria-label</NxCode> prop.
+        </NxP>
+      </NxTile.Subsection>
+      <NxTile.Subsection>
+        <NxTile.SubsectionHeader>
+          <NxH3>A Note about the Examples</NxH3>
+        </NxTile.SubsectionHeader>
+        <NxInfoAlert>
+          <NxCode>NxBreadcrumb</NxCode> should only be used with links that navigate within the current application
+          in a hierarchical manner. Using it in such a way here, within the RSC gallery, is not feasible as there
+          are not deep hiearchies within the organization of the gallery. As a result, the examples below have
+          all been constructed using links to external, third-party websites. In practice, this is not how they
+          would be used.
+        </NxInfoAlert>
+      </NxTile.Subsection>
+    </GalleryDescriptionTile>
+
+    <GalleryExampleTile title="Simple Example"
+                        id="nx-breadcrumb-simple-example"
+                        liveExample={NxBreadcrumbExample}
+                        codeExamples={exampleCode}>
+      A simple example of <NxCode>NxBreadcrumb</NxCode> showing three path segments with short names.
+    </GalleryExampleTile>
+
+    <GalleryExampleTile title="Example with Long Segments"
+                        id="nx-breadcrumb-long-segments-example"
+                        liveExample={NxBreadcrumbLongSegmentsExample}
+                        codeExamples={longSegmentsExampleCode}>
+      An example of <NxCode>NxBreadcrumb</NxCode> showing three path segments with long names which will truncate
+      proportionally such that the component maintains its default maximum width.
+    </GalleryExampleTile>
+
+    <GalleryExampleTile title="Example with Constrained Width"
+                        id="nx-breadcrumb-constrained-width-example"
+                        liveExample={NxBreadcrumbConstrainedWidthExample}
+                        codeExamples={constrainedWidthExampleCode}>
+      An example of <NxCode>NxBreadcrumb</NxCode> within a container that is narrower
+      than <NxCode>NxBreadcrumb</NxCode>'s maximum width. <NxCode>NxBreadcrumb</NxCode> shrinks to fit within the
+      container, and the truncation of the longs adjusts accordingly.
```

Comment:
```suggestion
      container, and the truncation of the long names adjust accordingly.
```

Reply:
@JenniKobylski just had me change how the truncation works. I fixed this typo as part of the corresponding doc update. 90768ca35